go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -33,6 +33,8 @@ class GoModUpdater
           # Package pseudo-version does not match the version-control metadata
           # https://golang.google.cn/doc/go1.13#version-validation
           /go: .*: invalid pseudo-version/m.freeze,
+          # Package has been transferred and the original location is now private (mholt/caddy)
+          /go: .*: invalid version/m.freeze,
```

Comment:
You can also get an invalid version message if you specify a version that doesn't exist but that seems in line with the errors around not finding the package in the repo.

```
go get github.com/caddyserver/caddy/v2@v2.9.9
go get: github.com/caddyserver/caddy/v2@v2.9.9: invalid version: unknown revision v2.9.9
```